tests/specs/api/stripe.spec.js

```diff
@@ -0,0 +1,77 @@
+import * as httpMock from '../../mocks/http';
+import { getAuthenticationHeaders } from '../../../src/utils/auth';
+import { StripeApi } from '../../../src/api/stripe';
+
+
+describe('Stripe API', () => {
+  const serviceUrl = 'http://some-url.com';
+  const userId = 'user-id';
+  const httpHeaders = getAuthenticationHeaders({
+    appToken: 'token'
+  });
+  let httpSpy;
+  let api;
+
+  beforeEach(() => {
+    httpSpy = httpMock.mock();
+    api = new StripeApi(serviceUrl, httpHeaders);
+  });
+
+  afterEach(() => {
+    httpMock.restore();
+  });
+
+  describe('#updateCustomer', () => {
```

Comment:
`POST /customer` is a jwt-only API, but your test is using `appToken`. Lib should probably return an error if app token is the only provided auth method when you attempt to hit this endpoint